src/components/List/ListHeader/_list-header.scss

```diff
@@ -0,0 +1,44 @@
+.list-header {
```

Comment:
we have added an IoT prefix that we should use to differentiate our styles from Carbon and the consuming application. 

https://github.com/IBM/carbon-addons-iot-react/blob/master/src/globals/_vars.scss

https://github.com/IBM/carbon-addons-iot-react/blob/caa120186687098c100639e226cc9c3cceb59449/src/components/Button/Button.jsx#L9

Reply:
hey, cool!  didn't know this.